.github/workflows/generate_llms_txt.yml

```diff
@@ -0,0 +1,30 @@
+name: Generate LLMs.txt File
+
+on:
+  # Run on push to main branch
+  push:
+    branches: [main]
+  # Allow manual triggering
+  workflow_dispatch:
+
+jobs:
+  generate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 18
+          
+      - name: Generate llms.txt
+        run: npx -y llmstxt gen https://docs.celestia.org/sitemap.xml > public/llms.txt
```

Comment:
I realized this is using the site to build it, so wouldn't build it from the new content being changed/added. so maybe this should be a cron job 🤔

Reply:
https://github.com/celestiaorg/docs/pull/2032/commits/ae4af0ebf9bd487aed9244e3c964f241229006ff

Reply:
I think post-merge would be even better:

<img width="818" alt="image" src="https://github.com/user-attachments/assets/031d3282-8b17-4df8-ae1e-1e1eae3e7129" />

```yaml
on:
  push:
    branches:
      - main  # or 'master', whatever your default branch is
    paths:
      - docs/**       # adjust these patterns to only match relevant changes
      - .github/workflows/generate_llms_txt.yml

```

Reply:
i think `/docs` doesn't exist. would the post-merge work still if the site hasn't built, since it's generating it from the built site?

Reply:
I hadn't noticed/focused on it. I think you are correct. 
In this case, removing this "paths" key altogether should be fine, really, because the cmd is reading from the deployed website.

Reply:
Also, I don't know how deployment is working for now, but I am guessing it takes a moment for it to be live after merge, right? 

So I got 2 decent solutions to ponder: 
<img width="828" alt="image" src="https://github.com/user-attachments/assets/aef6f950-b15f-466f-9b17-abd4929e954e" />
or 

<img width="876" alt="image" src="https://github.com/user-attachments/assets/4b600e2a-22b6-4a12-8cc1-46837f42a9e8" />

Reply:
let's do sleep for 10 mins then build it, should only take 5 mins max to build but better to be on safe side